test/integration/getEntries.test.ts

```diff
@@ -410,7 +458,21 @@ function assertLocalizedEntriesResponse(response) {
   expect(response.includes!.Asset).toBeDefined()
   // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
   expect(Object.keys(response.includes!.Asset!).length).toBeGreaterThan(0)
+
   expect(response.items[0].fields.bestFriend['en-US'].fields).toBeDefined()
   expect(response.items[0].fields.bestFriend['en-US'].sys.type).toBe('Entry')
   expect(response.items[0].metadata).toEqual({ tags: [] })
 }
+
+function assertCSMEntriesResponse(response) {
+  expect(response.includes).toBeDefined()
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  expect(response.includes!.Asset).toBeDefined()
```

Comment:
Nit: Are these non-null assertions really needed? Why not use `response.includes?.Asset` ?